DISPATCH-1338: Restructure Dispatch Router book for edge router (#512)
apache/qpid-dispatch: docs/books/user-guide/modules/integrating-with-kerberos.adoc

* In the Kerberos environment, a service principal of `amqp/<hostname>@<realm>` must be configured.
+
This is the service principal that {RouterName} uses.

* The `cyrus-sasl-gssapi` package must be installed on each client and the router host machine.

.Procedure

. On the router's host machine, open the `/etc/sasl2/qdrouterd.conf` configuration file.
+
--
.An `/etc/sasl2/qdrouterd.conf` Configuration File
====
[options="nowrap"]
----
pwcheck_method: auxprop
auxprop_plugin: sasldb
sasldb_path: qdrouterd.sasldb
keytab: /etc/krb5.keytab
mech_list: ANONYMOUS DIGEST-MD5 EXTERNAL PLAIN GSSAPI
----
====
--

. Verify the following:
** The `mech_list` attribute contains the `GSSAPI` mechanism.
** The `keytab` attribute points to the location of the keytab file.

include::{FragmentDir}/fragment-router-open-config-file-step.adoc[]

. In the `router` section, specify the path to the SASL configuration file.
+
--
[options="nowrap",subs="+quotes"]
----
router {
    mode: interior
    id: Router.A
    saslConfigDir: /etc/sasl2/
}
----
`saslConfigDir`:: The absolute path to the SASL configuration file that contains the path to the SASL database.
--

> . For each incoming connection that should use Kerberos for authentication, set the `listener` to use the `GSSAPI` mechanism.

Reviewer: ```suggestion
. For each incoming connection using Kerberos for authentication, set the `listener` to use the `GSSAPI` mechanism.
```
